react/features/base/styles/functions.js

```diff
@@ -6,6 +6,17 @@ import { ColorPalette } from './components';
 declare type StyleSheet = Object;
 export type StyleType = StyleSheet | Array<StyleSheet>;
 
+/**
+ * RegExp pattern for HEX color format.
+ */
+const HEX_COLOR_FORMAT
+    = /^#([0-9A-F]{2,2})([0-9A-F]{2,2})([0-9A-F]{2,2})$/i;
```

Comment:
This doesn't allow the short format of 3 chars, right? Maybe it should?

Reply:
Yes, because to support that I needed a special hex rounding function to be implemented in ```_getAlphaInHex``` so I decided not to implement it until we actually need that feature.

Reply:
Hum, isn't the 3 char version just a shorthand for when the values are the same? As in "abc" -> "aabbcc" and so on.

Reply:
Yes, but then you need a rounding function to round alpha values of e.g. '13' into '11' or 'AF' into 'BB' which is not that straightforward and I don't think we need it now.

Reply:
Sorry, I don't understand. If you accept "#ABC", then convent it into "#AABBCC" and then do the same as you do now, would it be wrong?